test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/PreallocatedTagHelperAttributes.cshtml

```diff
@@ -0,0 +1,3 @@
+@addTagHelper *, TestAssembly
+
+<input type="text" prefix-hello="world" prefix-test="value" />
```

Comment:
We already have a preallocated test. Try adding this to that test (if it's not already duplicated).